src/utils/findPositionOfCapturingGroup.js

```diff
@@ -0,0 +1,35 @@
+import PLACEHOLDERS from './placeholders'
+
+const findPositionOfCapturingGroup = (markup, parameterName) => {
+  if (parameterName !== 'id' && parameterName !== 'display') {
+    throw new Error("parameterName must be 'id' or 'display'")
```

Comment:
You could include what the `parameterName` actually was in the error message. This might make it easier for developers to spot the place where they made a mistake.